README.md

```diff
@@ -469,3 +469,64 @@ We support the latest versions of the following browsers:
 - Firefox
 
 IE 11 and earlier is not supported.
+
+## Frontend Packages
+- [ceph-storage-plugin](https://github.com/openshift/console/tree/master/frontend/packages/ceph-storage-plugin/)
```

Comment:
They are fully qualified because they are meant to link to the source code and not the README. Some packages don't have READMEs, but I still wanted to provide a link to something. Also, github pages is deploying from our master branch and deploys changes immediately, so we shouldn't have any issue with broken links as long as we update the README when a package leaves the repo.

Reply:
We most definitely will have broken links with this model 🤔 

Release 4.11 => New Pages with "master" link
Release 4.12 => New Package comes along, new "master" link
Release 4.13 => pipelines-plugin goes Dynamic and deletes their references in OpenShift Console; Breaks 4.11 and 4.12 release branches, their link will go to a 404 page.

Relative links do not need to go to a file, they can go to a folder too. An example (click the top link): https://github.com/andrewballantyne/code-examples/blob/test-github-path/README.md

Reply:
Ah -- right you are. https://andrewballantyne.github.io/code-examples does not work without a readme at the other end. 🤔

Reply:
I stubbed an empty readme and it worked again. Okay... my recommendation is not to highlight packages that do not want to describe themselves.

I see nothing but a nightmare of maintenance by adding branch names.

Reply:
I see what you mean. Hmm, okay, maybe we only provide links to packages with readme's then.